src/artifact_providers/github.ts

```diff
@@ -81,46 +82,80 @@ export class GithubArtifactProvider extends BaseArtifactProvider {
    */
   private async listArtifact(
     revision: string,
-    page = 0
+    revisionDate?: string,
+    page = 0,
+    tries = 0,
   ): Promise<ArtifactItem> {
-    const { repoName, repoOwner } = this.config;
+    const { repoName: repo, repoOwner: owner } = this.config;
     const per_page = 100;
 
     // https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#artifacts
     const artifactResponse = ((
       await this.github.request('GET /repos/{owner}/{repo}/actions/artifacts', {
-        owner: repoOwner,
-        repo: repoName,
+        owner: owner,
+        repo: repo,
```

Comment:
I'm wondering how prettier doesn't reformat this.

```suggestion
        owner,
        repo,
```

Reply:
:(

Will do in a follow up.